teuthology/suite/util.py

```diff
@@ -404,14 +404,14 @@ def has_packages_for_distro(sha1, os_type, os_version, flavor,
                             package_versions=None):
     """
     Checks to see if gitbuilder has packages for the given sha1, os_type and
-    kernel_flavor.
+    flavor.
 
     See above for package_versions description.
 
     :param sha1:             The sha1 hash of the ceph version.
     :param os_type:          The distro we want to get packages for, given
                              the ceph sha1. Ex. 'ubuntu', 'rhel', etc.
-    :param kernel_flavor:    The kernel flavor
+    :param flavor:           The distro flavor
```

Comment:
Following your nomenclature for `-f` option:
```suggestion
    :param flavor:           The ceph packages shaman flavor
```